packages/patternfly-4/react-core/src/components/Select/SelectOption.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import styles from '@patternfly/patternfly/components/Select/select.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import { SelectContext, KeyTypes } from './selectConstants';
+
+const propTypes = {
+  /** the value for the option */
+  children: PropTypes.string,
+  /** additional classes added to the Select Option */
+  className: PropTypes.string,
+  /** the value for the option */
+  value: PropTypes.string,
+  /** flag indicating if the option is disabled */
+  isDisabled: PropTypes.bool,
+  /** flag indicating if the option acts as a placeholder */
+  isPlaceholder: PropTypes.bool,
+  /** Optional on click callback */
+  onClick: PropTypes.func,
+  /** Callback for ref tracking */
+  sendRef: PropTypes.func,
+  /** Additional props are spread to the container <button> */
+  '': PropTypes.any
+};
+
+const defaultProps = {
+  children: null,
+  className: '',
+  value: null,
+  isDisabled: false,
+  isPlaceholder: false,
+  onClick: Function.prototype
+};
+
+class SelectOption extends React.Component {
+  ref = React.createRef();
+
+  componentDidMount() {
+    this.props.sendRef(this.ref.current, this.props.index);
+  }
+
+  onKeyDown = event => {
+    if (event.key === KeyTypes.Tab) return;
+    event.preventDefault();
+    if (event.key === KeyTypes.ArrowUp) {
+      this.props.keyHandler(this.props.index, 'up');
+    } else if (event.key === 'ArrowDown') {
+      this.props.keyHandler(this.props.index, 'down');
+    } else if (event.key === KeyTypes.Enter) {
+      this.ref.current.click && this.ref.current.click();
+    }
+  };
+
+  render() {
+    const {
+      children,
+      className,
+      value,
+      onClick,
+      isDisabled,
+      isPlaceholder,
+      selected,
```

Comment:
Linting error on selected.

Reply:
Running yarn lint isn't throwing me any errors. Am I running it incorrectly? Could you post the specific error?

Reply:
I've added the missing props and given them defaults. Let me know if its still throwing errors.